lib/service/invoicesubscription.go

```diff
@@ -243,11 +243,6 @@ func (svc *LndhubService) InvoiceUpdateSubscription(ctx context.Context) error {
 			if err != nil {
 				svc.Logger.Errorf("Error processing invoice update subscription: %v", err)
 				sentry.CaptureException(err)
-				// TODO: close the stream somehoe before retrying?
```

Comment:
I needed to remove this because the "error" that is returned here, if there is one, is most likely the context cancellation. Even if it is some kind of other error, I feel it doesn't really make sense to add these kind of ad-hoc reconnects here.